typescript/sdk/src/utils/cosmos.ts

```diff
@@ -274,7 +274,15 @@ export const CosmosChainSchema = z
               .object({
                 primary_color_hex: z
                   .string()
-                  .regex(new RegExp('^#[0-9a-fA-F]{6}$'))
+                  .min(1)
```

Comment:
This feels like it's likely to keep breaking. Do we definitely need this in the monorepo?

Reply:
I'll wait til post-hackathon to chat to trevor about this, as I believe he added it a while back in need of parsing some of these fields in the cosmos registry

Reply:
Maybe theres a way to import this via npm from the cosmos registry?

If not, we just need the gas price stuff really. So if we'd prefer to modify this to be a subset of the full spec, that works for me too